src/cli/cli.cpp

```diff
@@ -7682,6 +7699,70 @@ template <> otError Interpreter::Process<Cmd("txpower")>(Arg aArgs[])
     return error;
 }
 
+template <> otError Interpreter::Process<Cmd("debug")>(Arg aArgs[])
```

Comment:
Should we add the cli test to the `./tests/scripts/expect/`?

Reply:
Not sure. This is only intended for debugging and not a command we rely and use for operation of network.

I may add a simple `expect` check to verify that it emits `Done` (but this won't check much beyond the existence of the command). I don't think checking the full output will be useful as the list of emitted commands from `debug` may get changed in future and also depend on the set of config (which features are enabled).

Reply:
Added `cli-debug.exp` in new push (which just checks for "Done")/